package.json

```diff
@@ -17,7 +17,9 @@
     "displayName": "OlympusDAO Front End Tests",
     "collectCoverageFrom": [
       "src/**/*.{js,jsx,ts,tsx}",
-      "!<rootDir>/node_modules/"
+      "!<rootDir>/node_modules/",
+      "!src/locales/translations/**",
+      "!src/typechain/**"
     ],
     "coverageThreshold": {
       "global": {
```

Comment:
Should we update these thresholds back to the current level to prevent regressions?